app/models/league.rb

```diff
@@ -0,0 +1,9 @@
+class League < ActiveRecord::Base
+  validates :name, presence: true, uniqueness: true
```

Comment:
Remove the unique constraint - otherwise the same league can't draft again next year. With the passwords it's unnecessary anyway.